kairon/shared/pyscript/callback_pyscript_utils.py

```
@@ -571,13 +571,18 @@ def process_instruction(data_list, prompt, operation_type, model_id, llm_type: s
 
 
     @staticmethod
-    def create_vector_collection(collection_name, model_id: str, user: str, emb_size: int = 3072,
+    def create_vector_collection(collection_name, model_id: str, user: str,
                                  overwrite: bool = False, metadata: list = None, bot: str = None):
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)